src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -337,7 +337,7 @@ private OperationStatus CheckActiveStatements(IEnumerable<StatementSyntax> state
 
                 foreach (var statement in statements)
                 {
-                    if (!(statement is LocalDeclarationStatementSyntax declarationStatement))
+                    if (!(statement is LocalDeclarationStatementSyntax declarationStatement) || declarationStatement.Declaration.Variables.FullSpan.IsEmpty)
```

Comment:
Don't know if it is ever the case in practice, but it looks like the Declaration's variables allows null - http://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Generated/Syntax.xml.Internal.Generated.cs,aef1ed83774e9d67

Reply:
I take it back, the property is a struct, I didn't read closely enough :(